### PR TITLE
add SHGetStockIconInfo from shell32

### PR DESCRIPTION
**Pull request by tmm1**
provides access to many stock icons

### CODE REVIEW

**Comment by lxn** [comment]
Thanks and sorry for the delayed response.

Before I merge your changes, please add yourself to the AUTHORS file.

**Comment by tmm1** [comment]
I have added myself to the authors file.

File: shell32.go [hunk 1]
```diff
@@ -202,6 +321,7 @@ func init() {
 	shGetPathFromIDList = MustGetProcAddress(libshell32, "SHGetPathFromIDListW")
 	shGetSpecialFolderPath = MustGetProcAddress(libshell32, "SHGetSpecialFolderPathW")
 	shParseDisplayName = MustGetProcAddress(libshell32, "SHParseDisplayName")
+	shGetStockIconInfo = MustGetProcAddress(libshell32, "SHGetStockIconInfo")
```
Comment (tmm1) [inline shell32.go]: This was also added in Vista. What do you think about adding a new function `MaybeGetProcesAddress`?

Comment (lxn) [reply shell32.go]: I'd be ok with that.

**Pull request merged**